ipc/README.md

```diff
@@ -9,18 +9,16 @@ IPC server (Windows & Linux) for JSON-RPC 2.0.
 
 ```
 [dependencies]
-jsonrpc-core = "6.0"
-jsonrpc-ipc-server = "6.0"
+jsonrpc-ipc-server = { git = "https://github.com/paritytech/jsonrpc" }
```

Comment:
Is it okay to fetch git dependency here too?

Reply:
yup!